akka-docs/src/main/paradox/stream/operators/Flow/completionStageFlow.md

```diff
@@ -8,8 +8,8 @@ Streams the elements through the given future flow once it successfully complete
 
 ## Signature
 
-@@signature [Flow.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala) { #futureFlow }
-
+@apidoc[completionStageFlow](Flow$) { java="#completionStageFlow(java.util.concurrent.CompletionStage)" }
+@apidoc[completionStageFlow](Flow) { java="#completionStageFlow(java.util.concurrent.CompletionStage)" }
 @@@
```

Comment:
I want the auto replacement to remove the `@@@div { .group-scala }`/`@@@` too. I'll work on that soon.

Bonus points: the `@@@div { .group-scala }`/`@@@` filter is wrong on this particular oeprator since it's a java-specific operator.